plugins/modules/ome_template_identity_pool.py

```diff
@@ -174,18 +175,18 @@ def main():
             if template["IdentityPoolId"] == identity_id:
                 module.exit_json(changed=False, msg=NO_CHANGES_FOUND)
             if module.check_mode:
-                module.exit_json(changed=True, msg=CHANGES_FOUND)
+                module.exit_json(changed=False, msg=CHANGES_FOUND)
```

Comment:
```suggestion
                module.exit_json(changed=True, msg=CHANGES_FOUND)
```

Reply:
For check_mode changed will be true.
Hence it will be `module.exit_json(changed=True, msg=CHANGES_FOUND)`